internal/client/cloudfoundry/client.go

```diff
@@ -515,6 +515,32 @@ func (cfc *CFClient) getV2ApplicationsByPage(page int) ([]CFApplication, int, er
 	return results, appResp.Pages, nil
 }
 
+func (cfc *CFClient) GetV2Orgs() ([]cfclient.Org, error) {
+	// NOTE: this is used by the OrgCollector
```

Comment:
I think we can remove these notes. The compiler would tell you if you dare to remove these methods 😛

Reply:
True, I'll remove them :)